themes/digital.gov/layouts/partials/core/get_related.html

```diff
@@ -119,13 +127,11 @@ <h3>Get Started Building</h3>
   {{- $src := .src -}}
   {{- with .item -}}
     {{ $path := "" }}{{ with .File }}{{ $path = .Path }}{{ else }}{{ $path = .Path }}{{ end }}
-    <div class="promo" data-edit-this="{{- $path -}}">
+    <li class="promo" data-edit-this="{{- $path -}}">
       <div>
-        <h4>
-          <a class="external-link" href="{{- .Params.source_url -}}?=dg" title="{{ .Title | markdownify -}}"><span>{{- .Title | markdownify -}}</span></a>
-        </h4>
-        <img src="{{- $src | relURL -}}" alt="" aria-hidden="true"/>
+        <a class="external-link" href="{{- .Params.source_url -}}?=dg" title="{{ .Title | markdownify -}}"><span>{{- .Title | markdownify -}}</span></a>
```

Comment:
The title is duplicate of the text inside the link. We should try to add a little more context.

Reply:
I've updated line 30 (communities) and 132 (services) to provide more context for the title. Now I'm thinking it should be an `aria-label`?

Reply:
I think the title is okay as long as it's different than the text inside.